### DISCUSSION TITLE
Docker support

### DISCUSSION THREAD

**Discussion by rga5321** [mmagnus/Pocket-Plus-Calibre-Plugin#39] [Ideas]
Hi there,

3 years ago I created a Docker image that wrapped around this recipe and some nodejs code. The purpose was to automate the e-book generation and sending with a single line:

`docker run --rm --env-file YOUR_ENV_FILE alvaroreig/pocket2kindle-amd64`

Having installed only docker (and the image) in the server. No calibre.

By that time the archive function failed pretty often, so the code also called pocket api AFTER creating the e-book to archive any remaining article.

I think this code could be easily adapted to the current recipe, and so simplify the process for using it in a server.

I can't do it right now due to lack of time, but I would be happy to transfer the repository and the travis project to anyone interested. Or it could be folded into this repository.

The nodejs code could probably be replaced for a simple python script to maintain everything in a single language. That and the Dockerfile, of course.

Anyway, I just wanted to inform of its existence to anyone interested.

https://github.com/alvaroreig/pocket2kindle

Regards,

**Comment by mizzunet** [comment]
This is exactly I was looking for. Thank you so much🤗
